src/topography.f90

```diff
@@ -443,6 +444,29 @@ subroutine topography_min_max_depth(this, vgrid, level)
 
   end subroutine topography_min_max_depth
 
+  !-------------------------------------------------------------------------
+  subroutine topography_fill_fraction(this, sea_area_fraction)
+    class(topography_t), intent(inout) :: this
+    real(real32), intent(in) :: sea_area_fraction
+
+    write(output_unit,'(a,f7.2)') "Filling cells that have a sea area fraction smaller than ", sea_area_fraction
+
+    if (any(this%frac < sea_area_fraction)) then
+      where (this%frac < sea_area_fraction)
+        this%depth = 0.0
+        this%frac = 0.0
+      end where
+
+      if (this%lakes_removed == 'yes') then
+        ! We might have created new lakes, so rerun deseas
+        this%lakes_removed = 'no'
```

Comment:
should this be 'no ' (3 chars)?

Reply:
In a case like this, spaces are automatically added to match the variable length.

Reply:
Note that this different than `this%lakes_removed(1:2) = 'no'`, which would effectively leave the third character unchanged.